scripts/agentic/reconcile_devin_wiki_access.py

```
@@ -0,0 +1,340 @@
+    if not isinstance(rows, list):
+        raise ReconcilerError("Devin installation lookup did not return an installation array")
+    installations: list[Installation] = []
+    for row in rows:
+        if not isinstance(row, dict) or row.get("app_slug") != DEVIN_APP_SLUG:
+            continue
+        account = row.get("account")
+        if not isinstance(account, dict) or not isinstance(account.get("login"), str):
+            raise ReconcilerError("Devin installation did not include its target account")
+        selection = row.get("repository_selection")
+        if selection not in {"all", "selected"}:
+            raise ReconcilerError("Devin installation did not include a supported repository selection")
```

Pylint (4.0.6)

[convention] Line too long (104/100)

(C0301)